x-pack/plugins/security_solution/public/attack_discovery/pages/no_alerts/index.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AssistantAvatar } from '@kbn/elastic-assistant';
+import {
+  EuiEmptyPrompt,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiSpacer,
+  EuiText,
+} from '@elastic/eui';
+import React, { useMemo } from 'react';
+
+import * as i18n from './translations';
+
+const NoAlertsComponent: React.FC = () => {
+  const title = useMemo(
+    () => (
+      <EuiFlexGroup
+        alignItems="center"
+        data-test-subj="emptyPromptTitleContainer"
+        direction="column"
+        gutterSize="none"
+      >
+        <EuiFlexItem data-test-subj="emptyPromptAvatar" grow={false}>
+          <AssistantAvatar size="m" />
+          <EuiSpacer size="m" />
+        </EuiFlexItem>
+
+        <EuiFlexItem grow={false}>
+          <EuiFlexGroup alignItems="center" direction="row" gutterSize="none">
+            <EuiFlexItem data-test-subj="noAlertsTitle" grow={false}>
+              <span>{i18n.NO_ALERTS_TO_ANALYZE}</span>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    ),
+    []
+  );
+
+  const body = useMemo(
+    () => (
+      <EuiFlexGroup
+        alignItems="center"
+        data-test-subj="bodyContainer"
+        direction="column"
+        gutterSize="none"
+      >
+        <EuiFlexItem grow={false}>
+          <EuiText color="subdued" data-test-subj="bodyText">
+            {i18n.ATTACK_DISCOVERY_ONLY}
+          </EuiText>
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    ),
+    []
+  );
+
+  return (
+    <EuiFlexGroup
+      alignItems="center"
+      data-test-subj="noAlerts"
+      direction="column"
+      gutterSize="none"
+    >
+      <EuiFlexItem data-test-subj="emptyPromptContainer" grow={false}>
+        <EuiEmptyPrompt body={body} title={title} />
+      </EuiFlexItem>
+
+      <EuiFlexItem grow={false}>
+        <EuiLink
+          external={true}
+          data-test-subj="learnMoreLink"
+          href="https://www.elastic.co/guide/en/security/master/attack-discovery.html"
```

Comment:
nit: Consider using `docLinks` ala: 

`const docLinks = useKibana().services.docLinks;`

or changing version to `current` such that this specific version docs are shown (instead of latest/next version).

Looks like we plumbed through `docLinks` in the assistant code too, but not necessary here since we're within SecuritySolution scope:

https://github.com/elastic/kibana/blob/b20ca748214302131ffd510528a4b6a0961b9fc9/x-pack/packages/kbn-elastic-assistant/impl/assistant/assistant_title/index.tsx#L57-L58

Reply:
`docLinks` for Attack discovery don't exist yet, so I implemented your alternative suggestion:  changing the links from `master` to `current` in this commit: https://github.com/elastic/kibana/pull/182904/commits/50f32bc29c11e1fd95620f1d42701aff6b22e304